platform/platform_linux.cpp

```diff
@@ -83,8 +85,10 @@ Platform::Platform()
   auto const homeDir = GetEnv("HOME");
   CHECK(homeDir, ("Can't retrieve home directory"));
 
-  // ~/.config/OMaps/
-  m_settingsDir = JoinPath(*homeDir, ".config", "OMaps");
+  // XDG config directory, usually ~/.config/OMaps/
```

Comment:
Did you test it? Does it return the proper path and also check XDG if it's different?

Reply:
Yes I did. It uses $XDG_CONFIG_HOME if the variable exists, falling back to ~/.config.